__tests__/api.test.js

```diff
@@ -70,9 +69,130 @@ describe('GET - /api/articles/:article_id', () => {
         );
       });
   });
-})
+
+  test('GET comment count- ', () => {
+    return request(app)
+      .get(`/api/articles/9`)
+      .expect(200)
+      .then((res) => {
+        expect(res.body.article).toEqual(
+          expect.objectContaining({ comment_count: expect.any(String) })
+        );
+        expect(res.body.article.comment_count).toBe('2');
+      });
+  });
+});
+
+describe("6. PATCH /api/articles/:article_id", () => {
+  test("responds with status 200 and specific article is updated", () => {
+    return request(app)
+      .patch("/api/articles/2")
+      .send({ inc_votes: 50 })
+      .expect(200)
+      .then((res) => {
+        expect(res.body.article).toEqual(
+          expect.objectContaining({
+            article_id: 2,
+            title: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number)
+          })
+        );
+      });
+  });
+  

```

Comment:
You have no error tests for this. These need to be in place. 

400 - invalid article id type
400 - invalid data type for inc_votes value/anything missing on the request body
404 - valid but none existing article_id